account_analytic_project/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Project and analytic account integration
+========================================
+
+Gives the option to create projects directly from the analytic accounts they
+should be linked to.
+
+Without this module, if you first create an analytic account and later on you
+decide to have a project as well for this account, the situation is very
+difficult to rectify.
+
```

Comment:
1. While it is possible to create a project for an analytic account in 8.0 by specifying that you would want to use tasks, this was not possible before 8.0, but more importantly this is really not user-friendly. A user that wants a project for an analytic account expects a button to create a project.
2. You might want to use projects, but not use tasks.